src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -280,8 +272,8 @@ public function getCsrfParamName(): string
     public function startApp(?\yii\log\Logger $logger = null): void
     {
         codecept_debug('Starting application');
-        $config = require($this->configFile);
-        if (!isset($config['class'])) {
+        $config = include $this->configFile;
```

Comment:
we are checking that this file exists and is valid somewhere, or should we
```php
    if (! file_exists($this->configFile)) {
        throw new \RuntimeException("Config file not found: ” . $this->configFile);
    }
```
just before?

Reply:
Nah; we only configure the connector directly from the module; where it is validated.
In a more clean code world we'd properly encapsulate it and make it not public or at least readonly etc.